clients/drpoints/tests/bb_12_instr_5_x86.templatex

```diff
@@ -0,0 +1,4 @@
+T:1:5 :2:5 :3:5 [ ]*
```

Comment:
What are the brackets for?  This is a regex, right?  So just ` *` to match zero or more spaces.